designsafe/apps/data/templates/data/data_depot.html

```diff
@@ -68,6 +68,14 @@
     {% else %}
     <meta name="identifier" content="{{doi}}">
     {% endif %}
+    <!-- Dublin Core Metatags -->
+    <meta name="DC.identifier" content="{{doi}}">
```

Comment:
I'm not sure of the use case for this or the requirements, however there are two things I noticed.
- Is this identifier supposed to return something else if there is no DOI at the project level? In some cases there will be no DOI for the project; it will have multiple DOIs on the entities published within it.
- The DC meta content variables do not seem track changes in the view from one publication to another, but the meta content from the tags with names like `citation_XXX` do track.